src/modules/github.cpp

```diff
@@ -24,9 +24,15 @@ namespace modules {
 
     if (m_formatter->has(TAG_LABEL)) {
       m_label = load_optional_label(m_conf, name(), TAG_LABEL, "Notifications: %notifications%");
+    }
+
+    m_formatter->add(FORMAT_OFFLINE, TAG_LABEL_OFFLINE, {TAG_LABEL_OFFLINE});
 
-      update_label(0);
+    if (m_formatter->has(TAG_LABEL_OFFLINE)) {
+      m_label_offline = load_optional_label(m_conf, name(), TAG_LABEL_OFFLINE, "Offline");
     }
+
+    update_label(0);
```

Comment:
I'm not sure if we should keep this line.
Indeed it prevents to display the label without applying the data but this isn't done in any module.
Maybe, we should address this globally rather that adding quick fixes in every modules.

Reply:
As that was already the case, perhaps this isn't the best pr to address changing it in? Other than that the issues you noted should be fixed.

Reply:
I think we can let this line for now. When I'll post a PR that solves these issues I'll remove it.